ultra/ultra/scenarios/common/distributions.py

```diff
@@ -284,43 +284,48 @@
                 "distribution": behavior_distribution,
                 "start_end_on_different_lanes_probability": 0.0,
                 "begin_time_init": {
-                    "func": poisson_init,
-                    "params": {},
+                    "func": basic_begin_time_init_func,
+                    "params": {"probability": prob_heavy},
                 },
                 "has_turn": True,
                 "deadlock_optimization": True,
             },  # blocking
             "south-east": {
                 "vehicles": 100,
-                "distribution": behavior_distribution,
+                "distribution": {
+                    "default": 0.80,
+                    "aggressive": 0.20,
+                    "cautious": 0.00,
+                    "blocker": 0.0,
+                },
                 "start_end_on_different_lanes_probability": 0.0,
                 "begin_time_init": {
                     "func": burst_begin_time_init_func,
                     "params": {
-                        "vehicle_cluster_size": (1, 2),
+                        "vehicle_cluster_size": (1, 1),
                         "time_between_cluster": (60, 80),
-                        "time_for_each_cluster": 5,
+                        "time_for_each_cluster": 1,
                     },
                 },
                 "has_turn": True,
                 "deadlock_optimization": False,
             },  # blocking
             "south-north": None,  # blocking
             "west-east": {
-                "vehicles": 1000,
+                "vehicles": 200,
                 "distribution": {
-                    "default": 0.7,
-                    "aggressive": 0.30,
+                    "default": 0.80,
+                    "aggressive": 0.20,
                     "cautious": 0.00,
-                    "blocker": 0.00,
+                    "blocker": 0.0,
                 },
                 "start_end_on_different_lanes_probability": 0.0,
                 "begin_time_init": {
                     "func": burst_begin_time_init_func,
                     "params": {
-                        "vehicle_cluster_size": (1, 4),
-                        "time_between_cluster": (25, 35),
-                        "time_for_each_cluster": 3,
+                        "vehicle_cluster_size": (2, 4),
+                        "time_between_cluster": (15, 30),
```

Comment:
I noticed all the other changes for this value in other intersection types and routes were `(15, 35)` instead of `(15, 30)`. Is there any reason this is different?